MediaBrowser.Api/UserLibrary/ItemsService.cs

```diff
@@ -197,29 +198,27 @@ private QueryResult<BaseItem> GetQueryResult(GetItems request, DtoOptions dtoOpt
                 request.ParentId = null;
             }
 
-            var item = string.IsNullOrEmpty(request.ParentId) ?
-                null :
-                _libraryManager.GetItemById(request.ParentId);
+            BaseItem item = null;
 
-            if (item == null)
+            if (!string.IsNullOrEmpty(request.ParentId))
             {
-                item = string.IsNullOrEmpty(request.ParentId) ?
-                    user == null ? _libraryManager.RootFolder : _libraryManager.GetUserRootFolder() :
-                    _libraryManager.GetItemById(request.ParentId);
+                item = _libraryManager.GetItemById(request.ParentId);
             }
 
-            // Default list type = children
+            if (item == null)
+            {
+                item = _libraryManager.GetUserRootFolder();
```

Comment:
What happens if `user` is `null`?

Reply:
Than it will crash, just as before.

Reply:
Previously there was a check for that case, now it's gone.

Reply:
There was, but a bit later it didn’t check before using it, so it still would have crashed.